db/models/address.go

```diff
@@ -2,7 +2,7 @@ package models
 
 type Address struct {
 	ID      uint   `gorm:"primary_key"`
-	Address string `gorm:"type:varchar(128); unique_index"`
+	Address string `gorm:"type:varchar(128); index:unique"`
```

Comment:
`index:unique` does this work or did previous version worked? I think it might need to be `sql:"index"` ?

Reply:
In other examples it was: https://github.com/trustwallet/blockatlas/pull/1218/files#diff-07e9c6898f34339b8de0bd692b136955R7

Reply:
> In other examples it was: https://github.com/trustwallet/blockatlas/pull/1218/files#diff-07e9c6898f34339b8de0bd692b136955R7

It is correct. We need to have an index. Also it is unique index. Gorm v2 has new type declarations

Reply:
Do you have a link to new type of declarations?

Reply:
I changed it to ```uniqueIndex``` which equal index, unique. 

Declaring Models doc:  https://gorm.io/docs/models.html